ng-dev/auth/shared/ng-dev-token.ts

```diff
@@ -107,16 +107,12 @@ async function saveTokenToFileSystem(data: NgDevUserWithToken) {
 
 /** Retrieve the token from the file system. */
 async function retrieveTokenFromFileSystem(): Promise<NgDevUserWithToken | null> {
-  try {
-    if (!(await stat(tokenPath))) {
-      return null;
-    }
-  } catch {
-    return null;
+  if (await hasEvidenceOfPreviousLogin()) {
```

Comment:
Super nit: I think it would be better to call the function `hasTokenStoreFile`. Only considering the name it doesn't seem to guarantee that the `tokenPath` exists.